install_script.sh.template

```diff
@@ -1098,11 +1098,11 @@ elif [ ! "$no_agent" ]; then
     if [ -z "$fips_mode" ]; then
       if [ -n "$site" ]; then
         printf "\033[34m\n* Setting SITE in the $nice_flavor configuration: $config_file\n\033[0m\n"
-        $sudo_cmd sh -c "sed -i 's/# site:.*/site: $site/' $config_file"
+        $sudo_cmd sh -c "sed -i 's/^# site:.*$/site: $site/' $config_file"
```

Comment:
this would be a good candidate to be extracted as a function

Reply:
it is, maybe you are reviewing an old commit :/

Reply:
ah indeed this is probably part of a review I did commit per commit.
Resolving